rstudio/rhel9-python-3.9/run-rstudio.sh

```diff
@@ -16,6 +16,7 @@ fi
 
 # Create lib folders if it does not exist
 mkdir -p  /opt/app-root/src/Rpackages/4.3
+cp -r /opt/app-root/bin/Rpackages/4.3/* /opt/app-root/src/Rpackages/4.3/
```

Comment:
What happens when user starts workbench, installs additional packages, waits few months and restarts the workbench? if this `cp` command runs then, won't it put `/opt/app-root/src/Rpackages/4.3/` into inconsistent state, mixing old and new versions of r packages?

Reply:
Yeah, good point.
perhaps, we should check, if the package already exists, we don't copy.
lets make this change.